app/models/book.rb

```diff
@@ -1,3 +1,3 @@
 class Book < ApplicationRecord
-  
+  has_many :reviews
```

Comment:
to really complete this relationship, would like to see the through relationship. 
```
has_many :reviews
has_many :users, through: :books
```
This would allow us to have a way to access people who have reviewed each book by calling `book.users`